cookbooks/swift/recipes/source.rb

```diff
@@ -80,35 +80,21 @@
 execute "python-pyeclib-install" do
   cwd "#{node['source_root']}/pyeclib"
   command "pip install -e . && pip install -r test-requirements.txt"
-  if not node['full_reprovision']
-    creates "/usr/local/lib/python2.7/dist-packages/pyeclib.egg-link"
```

Comment:
We want to do this every time now? Does anything still look at `full_reprovision`? What's the distinction supposed to be these days?

Reply:
I'm giving up on full_reprovision - the idea was basically to allow these "creates" vars to speed up reinstall, but realizing that it hasn't been working on py3 environments and it's not actually a problem for me I decided it should go as we remove py2 and we can reintroduce it later if we think that's even possible.